llvm/include/llvm/Object/OffloadBinary.h

```diff
@@ -47,6 +47,7 @@ enum ImageKind : uint16_t {
   IMG_Cubin,
   IMG_Fatbinary,
   IMG_PTX,
+  IMG_SPIRV,
```

Comment:
@asudarsa.
ImageKind and OffloadKind are being encoded in the OffloadWrapper and this is going to be a part of ABI.
I see that you have changed OffloadKind to powers of 2. We need to settle some consistency here.